lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -269,6 +269,11 @@ private JourneyResponse getJourneyResponse(
                             .getParsedVtr()
                             .getLowestStrengthRequestedGpg45Vot(configService);
 
+            // As almost all of our journeys are proving or mitigating a GPG45 vot we set the
+            // target vot here as a default value. It will be overridden for identity reuse.
```

Comment:
I think it might make more sense the other way round, where you're eliminating options rather than opening up.

Start broad, then if there's not a match, slim it down to gpg45...